api/app/jobs/env_canada.py

```diff
@@ -55,56 +55,56 @@ def parse_gdps_rdps_filename(filename):
     prediction_hour = last_part[0][1:]
     prediction_timestamp = model_run_timestamp + \
         datetime.timedelta(hours=int(prediction_hour))
-    return model, variable_name, projection, model_run_timestamp, prediction_timestamp
+    return variable_name, projection, model_run_timestamp, prediction_timestamp
 
 
-def parse_high_res_model_filename(filename):
+def parse_high_res_model_url(url):
```

Comment:
Can we write a regression test for this that would have caught the bug?

Reply:
Added positive and negative test cases for this. Should also raise Exception when the provided URL doesn't have as many parts as it should have (I know it's only very basic error-checking)